pom.xml

```diff
@@ -130,8 +130,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.3.2</version>
                 <configuration>
-                    <source>1.5</source>
-                    <target>1.5</target>
+                    <source>1.6</source>
```

Comment:
Hi @s2oBCN ! Your changes look great! 

All but this. I've pulled your pull request locally and re-build it leaving 1.5 instead of updating the Java version. It build successfully. 

Any specific reason for upgrading to 1.6?